### PR TITLE
standardize Makefile whitespace between platforms, a little tidy

### PR DESCRIPTION
**Pull request by markwkidd**

### CODE REVIEW

**Comment by grant2258** [comment]
removing -std=gnu90 you going to cause problems

File: Makefile [hunk 1]
```diff
@@ -545,17 +545,22 @@ else
...
 endif
 
-# Compiler Flags For All Platforms #############################
+# End of architecture-specific flags ######################
+
+# Compiler flags for all platforms #############################
+
+# we actually code to c89 but in order to allow // comments in c source, gnu90 is the closest standard
+# beware that gnu90 does still allow some code through that will not compile for msvc or emscripten though
+CFLAGS += -std=gnu90
```
Comment (markwkidd) [inline Makefile]: @grant2258 gnu90 is relocated here

Comment (markwkidd) [reply Makefile]: I guess that doesn't quite qualify as a tidy but. maybe "tiny refactor" 👾

**Comment by grant2258** [comment]
will that work in msvc ?

**Comment by markwkidd** [comment]
yes I think it is fine in msvc but actually not, as written, with the nintendo switch. let me think about how to fix my error

**Comment by markwkidd** [comment]
I'm going to see if switch actually needs gnu11 or not.. time to watch the buildbot

**Pull request merged**